src/freenas/etc/rc.freenas

```diff
@@ -90,3 +90,18 @@ ro_sqlite()
 	cp ${FREENAS_CONFIG} ${ret}
 	echo ${ret}
 }
+
+ha_hardware()
+{
+	/usr/local/bin/python3 $(find /usr/local -name hadetect.py -type f) | jq -r '.hardware'
```

Comment:
hadetect is an entry point in middlewared setup.py... It should be just available in /usr/local/bin instad of requiring us to use find every time?

Reply:
true